api-tests/server/platform_test.go

```diff
@@ -209,4 +209,34 @@ func TestPlatform(t *testing.T) {
 			require.NotNil(t, resp.GetPayload().Entitlements)
 		})
 	})
+
+	t.Run("get contact information", func(t *testing.T) { //nolint:dupl
+		if username == "" || password == "" {
+			t.Skip("Environment variables PERCONA_TEST_PORTAL_USERNAME, PERCONA_TEST_PORTAL_PASSWORD not set.")
+		}
+
+		t.Run("success", func(t *testing.T) {
+			_, err := client.Connect(&platform.ConnectParams{
+				Body: platform.ConnectBody{
+					ServerName: serverName,
+					Email:      username,
+					Password:   password,
+				},
+				Context: pmmapitests.Context,
+			})
+			require.NoError(t, err)
+
+			// Confirm we are connected to Portal.
+			settings, err := serverClient.GetSettings(nil)
+			require.NoError(t, err)
+			require.NotNil(t, settings)
+			assert.True(t, settings.GetPayload().Settings.ConnectedToPlatform)
+
+			resp, err := client.GetContactInformation(&platform.GetContactInformationParams{Context: pmmapitests.Context})
+			require.NoError(t, err)
+			require.NotEmpty(t, resp.GetPayload().CustomerSuccess.Email)
```

Comment:
should we check if email is equal to email used to connect?

Reply:
no this is not the same as the user's email. this is the email of the customer success employee that deals with the company that the percona platform user belongs to

Reply:
oh, ok, thanks for explanation